sycl/test/fpga_tests/fpga_ac_int.cpp

```diff
@@ -120,7 +120,7 @@ void test_sub(queue &Queue, const ac_int<W, S> a, const ac_int<W2, S2> b,
     auto x = inp1.template get_access<kSyclRead>(h);
     auto y = inp2.template get_access<kSyclRead>(h);
     auto res = result.template get_access<kSyclWrite>(h);
-    h.single_task<class add>([=]() { res[0] = x[0] - y[0]; });
+    h.single_task<class sub>([=]() { res[0] = x[0] - y[0]; });
```

Comment:
```suggestion
    h.single_task<class sub>([=] { res[0] = x[0] - y[0]; });
```

Reply:
Interesting @keryell.  This is a matter of style as far as I know - what makes you prefer no `()`?  There are arguments each way, but consistency seems better than random choice, and the `()` are needed as soon as an attribute is to be applied.  I don't think much thought has gone into this aspect of the style, though.

@rolandschulz mentioned that clang-format doesn't seem to enforce either style.  Should it?

Reply:
My point is to make SYCL less verbose by 2 or 3 characters in simple lambdas.
Doing this, I hope it is less traumatic for hard-core HLS C programmers... :-)

Reply:
But sometimes I am contradicting myself about brevity against C style compatibility... https://github.com/oneapi-src/oneAPI-samples/pull/254#discussion_r509788394

Reply:
If you think this is important, I would recommend to file a bug against clang-format. Whenever possible style should be enforceable by tooling. I did a quick search and couldn't find an open feature request for this.

Reply:
I do not think `clang-format` changes the code like that because it is outside of just formatting issue.
This is the work of `clang-tidy`.

Reply:
clang-format does do changes which are more than just spacing (e.g. sort-includes). But that is the exception and the most similar one (https://clang.llvm.org/extra/clang-tidy/checks/readability-braces-around-statements.html) is in clang-tidy. Therefore this might indeed be more appropriate for clang-tidy.

Independent of that: I suggest we don't enforce a rule until there is a tool for it.